com.github.horaciodrs.tradesim.json

```diff
@@ -0,0 +1,27 @@
+{
+    "app-id": "com.github.horaciodrs.tradesim",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "base":"io.elementary.BaseApp",
+    "base-version": "juno-19.08",
```

Comment:
This probably should be 20.08.

Reply:
I tested this json file with flatpak-build and is working. The base-version, I think is correct. The build log says problem is on appdata.xml

Reply:
No it's not. juno-19.08 is based on fdo 19.08 like gnome 3.36. 3.38 is based on top of fdo 20.08.

Reply:
Ok. I changed runtime-version from 3.38 to 3.36 and also change the "com.github.horaciodrs.TradeSim.appdata.xml" filename to lower case "com.github.horaciodrs.tradesim.appdata.xml"
Please you can try run build process to send me de feedback

Reply:
No it's not the runtime that you have to downgrade, but rather upgrade the base version.

Reply:
Ok, Thank you for your fast answer. I have changed version number according to you say. Its now correct?